### PR TITLE
Provider exporter

### PR DESCRIPTION
**Pull request by javeedESyn**

### CODE REVIEW

File: src/Dfc.ProviderPortal.TribalExporter/Functions/ApprenticeshipDeltaExport.cs [hunk 1]
```diff
@@ -14,6 +14,7 @@ namespace Dfc.ProviderPortal.TribalExporter.Functions
 {
     public static class ApprenticeshipDeltaExport
     {
+        [Disable]
```
Comment (MrKevJoy) [inline src/Dfc.ProviderPortal.TribalExporter/Functions/ApprenticeshipDeltaExport.cs]: Hey Jav, do you need the functions disabled?

Comment (javeedESyn) [reply src/Dfc.ProviderPortal.TribalExporter/Functions/ApprenticeshipDeltaExport.cs]: good point ! no they need to be enabled
reject this one, I will request another one !

File: src/Dfc.ProviderPortal.TribalExporter/Functions/ProviderMigrator.cs [hunk 2]
```diff
@@ -0,0 +1,356 @@
...
+                    string input,  // Work around https://github.com/Azure/azure-functions-vs-build-sdk/issues/168
+                    ILogger log,
+                    [Inject] IConfigurationRoot configuration,
+                    [Inject] IProviderCollectionService providerCollectionService,
+                    [Inject] ICosmosDbHelper cosmosDbHelper,
+                    [Inject] IBlobStorageHelper blobhelper,
+                    [Inject] IUkrlpApiService ukrlpApiService
+                    )
+        {
+            const string WHITE_LIST_FILE = "ProviderWhiteList-Jav.txt";
```
Comment (MrKevJoy) [inline src/Dfc.ProviderPortal.TribalExporter/Functions/ProviderMigrator.cs]: Hey Jav, I think we should be using the same whitelist file across the board.

Comment (javeedESyn) [reply src/Dfc.ProviderPortal.TribalExporter/Functions/ProviderMigrator.cs]: Yes noticed that ! have changed it already :)

File: src/Dfc.ProviderPortal.TribalExporter/Functions/ProviderMigrator.cs [hunk 3]
```diff
@@ -0,0 +1,356 @@
...
+                    providerVerificationdetails.Add(new Verificationdetail()
+                    {
+                        VerificationAuthority = providerVerificationDetail.VerificationAuthority,
+                        VerificationID = providerVerificationDetail.VerificationID,
+                    });
+                }
+
+                Provider providerToUpsert = new Provider(providercontacts.ToArray(), provideraliases.ToArray(), providerVerificationdetails.ToArray());
+
+                // TODO : providerToUpsert.ProviderId = tribalData.ProviderId;
```
Comment (MrKevJoy) [inline src/Dfc.ProviderPortal.TribalExporter/Functions/ProviderMigrator.cs]: Should this be commented out?

**Pull request closed without merging**